mapping_parenting_tech/analysis/prototyping/play_store/fn_tester.py

```diff
@@ -0,0 +1,140 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.7
+#   kernelspec:
+#     display_name: Python 3.8.0 ('mapping_parenting_tech')
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Testing notebook
+# Testing refactoring of functions to manage ids, details and reviews of Play Store apps
```

Comment:
Could you please add a sentence about how you would expect someone to use this notebook? Or do you actually intend anyone to use it at all (perhaps it was only for when you were refactoring the functions?)

Reply:
Note: This notebook was only intended to be used as part of the refactoring checks.